openlcs/libs/tests.py

```diff
@@ -546,3 +546,32 @@ def test_golang_unsupported_extension(self):
     def test_yarn(self):
         # For now, npm parser is used for yarn as well.
         pass
+
+
+class TestGetTaskRepository(TestCase):
+
+    def setUp(self):
+        os.environ.setdefault(
+                'DJANGO_SETTINGS_MODULE', 'openlcs.openlcs.settings')
+        # Input the test source container build
+        self.test_package_nvr = 'ubi8-micro-container-source-8.7-1.1'
+        self.test_task_id = 48863576
+        self.context = {'config': {'KOJI_DOWNLOAD': settings.KOJI_DOWNLOAD,
+                                   'KOJI_WEBSERVICE': settings.KOJI_WEBSERVICE,
+                                   'KOJI_WEBURL': settings.KOJI_WEBURL
+                                   }
+                        }
+        # The expected source container repository
+        self.repository = 'registry-proxy.engineering.redhat.com/rh-osbs/ubi8-ubi-micro:rhel-8.7.0-containers-candidate-68432-20221108135839-x86_64' # noqa
```

Comment:
not sure if it's an internal url though. if it's confidencial , it should not be exposed.

Reply:
Updated as setting it as an environment variable.